protobuf/avs.proto

```diff
@@ -676,6 +684,9 @@ service Aggregator {
   // Smart Acccount Operation
   rpc GetNonce(NonceRequest)        returns (NonceResp) {};
   rpc GetWallet(GetWalletReq)       returns (GetWalletResp) {};
+  rpc HideWallet(GetWalletReq)      returns (GetWalletResp) {};
+  rpc UnhideWallet(GetWalletReq)    returns (GetWalletResp) {};
+  rpc SetWallet(SetWalletReq)       returns (GetWalletResp) {};
```

Comment:
let’s use a single SetWallet as the interface, and remove the HideWallet and UnhideWallet. 

The SetWalletReq can keep the same. The client side can use this method to set is_hidden to true or false.